### ISSUE TITLE
Setting unknown property: yii\widgets\ActiveField::inputTemplate

### ISSUE THREAD

**Issue by havryliv** [andrewblake1/yii2-credit-card#1]
Issue file: **andrewblake1\yii2-credit-card\CreditCard.php**
Steps to reproduce: 
1. install plugin & setup **ActiveForm**
2. run

**ActiveField version 2.0**

Solved by: changed provided for line No **86** - **inputTemplate** to **template**

P.S. Sorry for my English.

**Comment by andrewblake1** [comment]
Your english is fine - I only speak 1 language.

I have been using the extension in a different way but hadn't altered the documentation since no-one until now had used it except for me. I have altered the documentation and extension now to this as it is a bit more in the Yii fashion. Unfortunately this may break your existing usage as it removes the need for CreditCard.php altogether.

I have added you as a collaborator. Feel free to make changes as you see necessary - though may not be necessary if using as per the updated docs
